cmd/kube-controller-manager/app/core.go

```diff
@@ -632,3 +649,15 @@ func getNodeCIDRMaskSizes(clusterCIDRs []*net.IPNet, maskSizeIPv4, maskSizeIPv6
 	}
 	return nodeMaskCIDRs
 }
+
+func parseControllerDialhostIPDenylist(ctx *ControllerContext) ([]*net.IPNet, error) {
```

Comment:
nit: capitalize host
```suggestion
func parseControllerDialHostIPDenylist(ctx *ControllerContext) ([]*net.IPNet, error) {
```

Reply:
done